<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSDialog</class>
 <widget class="QDialog" name="TSDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabu Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tabu list volume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> iterations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>   Shake each</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TSDialog</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>TSDialog</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>TSDialog</receiver>
   <slot>slot4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>run()</slot>
  <slot>draw_metamodels()</slot>
  <slot>use_metamodels()</slot>
  <slot>loadAlgConf()</slot>
  <slot>openDialog()</slot>
  <slot>change_metamodel()</slot>
  <slot>changePCPersent()</slot>
  <slot>showMetamodelsRes()</slot>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
  <slot>slot4()</slot>
 </slots>
</ui>
